### DISCUSSION TITLE
Progressive drop down menu in UI

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1255] [Help]
Hi is there a way to have a progressive category drop down menu when entering data . for example in the picture below  in the number 1 column if I select unit streams in a drop down menu how can I make it so the second drop down input column 2 only show me the the 4 in blue (crude unit, solvent unit, DHT, unif/plat unit). Then when selecting one of those the third drop down menu only show me the 3rd group of selection?  is this posable ?
![image](https://user-images.githubusercontent.com/80058238/110140477-fdae1280-7d88-11eb-90bb-0e2752a3eaa6.png)

**Comment by swing18** [comment]
did you ever figure this out?

**Reply by swing18** [reply]
https://github.com/Budibase/budibase/discussions/3983

This expains how to do it

**Reply by melohagan** [reply]
Great find; thanks for sharing!
